package.json

```diff
@@ -34,7 +33,7 @@
     "grunt-shell": "^4.0",
     "grunt-text-replace": "^0.4",
     "load-grunt-tasks": "~5.1",
-    "parcel": "^2.12",
+    "parcel": "1.12.3",
```

Comment:
@itambek-godaddy it looks like there's a v1.12.5 [available on npm](https://www.npmjs.com/package/parcel-bundler) under the `parcel-bundler` name. Should we be using that instead?

Reply:
Ah, right - it's because starting `1.12.4` there is [an issue with the build](https://github.com/parcel-bundler/parcel/issues/5943). The [workaround mentioned here](https://github.com/parcel-bundler/parcel/issues/5943#issuecomment-788928442) did not work, so I opted for downgrading instead.